magicctapipe/resources/config_general.yaml

```diff
@@ -0,0 +1,28 @@
+mc_tel_ids:
+    LST-1: 1
+    LST-2: 0
+    LST-3: 0
+    LST-4: 0
+    MAGIC-I: 2
+    MAGIC-II: 3
+
+directories:
+    workspace_dir : "/fefs/aswg/workspace/raniere/" 
+    scripts_dir   : "/fefs/aswg/workspace/raniere/"
```

Comment:
if you use console scripts like you implemented now, in principle `scripts_dir` is not needed anymore, because you can call the scripts from anywhere. So in principle even when submitting the jobs, if the environment is activated (which it is), then calling the console scripts should be enough, with no path attached. Probably to be tested. But that would avoid for the users to copy the scripts in their directory.

Reply:
Should be fixed (seems running)